tests/virtualenv_tools_test.py

```diff
@@ -207,3 +208,191 @@ def test_not_a_virtualenv_missing_site_packages(fake_venv, capsys):
         fake_venv, fake_venv.join('lib/python2.7/site-packages'),
     )
     assert out == expected
+
+# -------- BEGIN Temp PyPy support Tests ------------
+# After base functionality for pypy venv movement is reviewed
+# these can be refactored in a way similar to how it was done in
+# https://github.com/Yelp/virtualenv-tools/pull/11 to reduce duplication
+
+
+skip_test_if_requires_pypy3 = pytest.mark.skipif(
+    sys.version_info >= (3, 3),
+    reason='pypy3 is not supported.'
+)
```

Comment:
Clarified with @joshSzep. This allows us to punt on pypy3 support, since we don't have it installed our on dev machines, and actually don't care about it much. Pre-existing policy tests python3 when the test interpreter is python3, by analogy, we'd test pypy3 when test interpreter is python3, but we're punting for now, instead.